File: Facebook.NET - VB/Contracts/GraphAPI.Account.vb
Imports System
Imports System.Runtime.Serialization

Namespace Facebook
	Public NotInheritable Partial Class GraphApi
        <DataContract()>
        Partial Public NotInheritable Class Account
            <DataMember(Name:="data")>
            Public Property Data() As GraphApi.AccountData()
            <DataMember(Name:="paging")>
            Public Property Paging() As GraphApi.Pagination
        End Class
	End Class
End Namespace

File: Facebook.NET/Contracts/GraphAPI.Account.cs
using System;
using System.Runtime.Serialization;

namespace Facebook
{
    public static partial class GraphApi
    {
        [DataContract]
        public sealed partial class Account
        {
            [DataMember(Name = "data")]
            public GraphApi.AccountData[] Data { get; set; }

            [DataMember(Name = "paging")]
            public GraphApi.Pagination Paging { get; set; }
        }
    }
}